security/pkg/caclient/client.go

```diff
@@ -20,13 +20,18 @@ import (
 
 	"istio.io/istio/pkg/log"
 	"istio.io/istio/security/pkg/caclient/grpc"
-	"istio.io/istio/security/pkg/pki/util"
+	pkiutil "istio.io/istio/security/pkg/pki/util"
 	"istio.io/istio/security/pkg/platform"
 	pb "istio.io/istio/security/proto"
 )
 
+// CAClient is a client to provision key and certificate from the upstream server.
+type CAClient interface {
+	RetrieveNewKeyCert() (newCert []byte, certChain []byte, privateKey []byte, err error)
+}
+
 // The CAClient can wraps details of the CSR protocol.
-type CAClient struct {
+type cAClient struct {
```

Comment:
CAClient and cAClient might be confusing. CAClientImpl maybe?

Reply:
Sure.